params/computation_cost_params.go

```diff
@@ -182,4 +173,32 @@ const (
 	TstoreComputationCost      = 280
 	BlobHashComptationCost     = 165
 	BlobBaseFeeComputationCost = 120
+
+	// opcode computation cost modification - istanbul
+	AddmodComputationCostIstanbul = 1410
+	MulmodComputationCostIstanbul = 1760
+	ShlComputationCostIstanbul    = 478
+	ShrComputationCostIstanbul    = 498
+	SarComputationCostIstanbul    = 834
+	XorComputationCostIstanbul    = 454
+	NotComputationCostIstanbul    = 364
+
+	// opcode computation cost codification - cancun
+	SdivComputationCostCancun    = 360
+	ModComputationCostCancun     = 320
+	AddmodComputationCostCancun  = 360
+	MulmodComputationCostCancun  = 700
+	ExpComputationCostCancun     = 720
+	Sha3ComputationCostCancun    = 560
+	Mstore8ComputationCostCancun = 230
+
+	SloadComputationCostCancun  = 2550
+	SstoreComputationCostCancun = 2510
+	Log1ComputationCostCancun   = 500
+	Log2ComputationCostCancun   = 500
+	Log3ComputationCostCancun   = 500
+	Log4ComputationCostCancun   = 500
+
+	OpcodeComputationCostLimit       = 100000000 // 100ms
+	OpcodeComputationCostLimitCancun = 150000000 // 150ms
```

Comment:
```suggestion
	OpcodeComputationCostLimit       = uint64(100000000) // before cancun
	OpcodeComputationCostLimitCancun = uint64(150000000)
	OpcodeComputationCostLimitOverride = uint64(0) // set by --opcode-computation-cost-limit. Overrides chain default settings above.
	OpcodeComputationCostLimitInfinite = math.MaxUint64 // pass it to disable computation cost checks
```

Reply:
Aggregated the parameters in computation_cost_params.go https://github.com/klaytn/klaytn/commit/3709fd8a22b92ef8553e5d422e216d8fc0436e99